### ISSUE TITLE
Termux crashes were using VNC server

### ISSUE THREAD

**Issue by mistrysiddh** [infinyte7/Manjaro-Linux-on-Android#22]
i have install Manjaro on lates version of manjaro and when i launch vncserver my termux get crashes
![Screenshot_20220617-201938_Termux](https://user-images.githubusercontent.com/25217451/174322519-4d2d5ad8-01d1-4c71-b0d8-b22581670f0a.png)

**Comment by krmanik** [comment]
Which version of vncserver you are using?
May be vncserver need to downgraded to 1.10.1
https://github.com/krmanik/Manjaro-Linux-on-Android/issues/5

```
wget https://raw.githubusercontent.com/AndronixApp/AndronixOrigin/master/Pacman/tigervnc-fix.sh
chmod +x tigervnc-fix.sh
./tigervnc-fix.sh
```

**Comment by mrfoxie** [comment]
> Which version of vncserver you are using? May be vncserver need to downgraded to 1.10.1 #5
> 
> ```
> wget https://raw.githubusercontent.com/AndronixApp/AndronixOrigin/master/Pacman/tigervnc-fix.sh
> chmod +x tigervnc-fix.sh
> ./tigervnc-fix.sh
> ```

yes i used this script after that only my termux started acting like this

**Comment by krmanik** [comment]
Okay, Let me check it then.

**Comment by mrfoxie** [comment]
> Okay, Let me check it then.

sure

**Comment by morilp** [comment]
The problem is Phantom which is a update for Android 12 google it you can see more detail, but you can run below command with adb on you Windows or Linux after that completely close your termux and run it again:
(Only work on Android-12 and above)
adb shell "/system/bin/device_config put activity_manager max_phantom_processes 2147483647"
